tests/spread/integration/zstd/task.yaml

```diff
@@ -0,0 +1,26 @@
+summary: Integration tests for zstd
+
+execute: |
+  rootfs="$(install-slices base-files_bin bash_bins dash_bins grep_bins ncurses-base_terminfo \
+    zstd_bins less_bins zstd_scripts)"
+  
+  chroot "${rootfs}/" pzstd -h
+  chroot "${rootfs}/" unzstd -H
+  chroot "${rootfs}/" zstd -H
+  chroot "${rootfs}/" zstdcat -H
+  chroot "${rootfs}/" zstdless --version
+  chroot "${rootfs}/" zstdmt -H
```

Comment:
can you also please add a smoke test this this? it can be similar to `zstd`

Reply:
Hi @cjdcordeiro,

To clarify, you are referring to `zstdmt`?

Reply:
Yes. There's only the `-H` test, but we could also test some real functionality since it should provide a comparable output to `zstd`